test/integration/mgo_items_store.go

```diff
@@ -19,6 +19,7 @@ type (
 	// Store allows operations on items.
 	Store interface {
 		Create(i *Item) (*Item, error)
+		RemoveAll() error
```

Comment:
Can you add a FindBSONRaw implementation and add a test?

Reply:
will do!